dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -601,6 +608,16 @@ public void write(OutputStream os) throws IOException,
 
                                         zipper.addToManifest(fileName + " (" + mimeType + ") " + " skipped because the total size of the download bundle exceeded the limit of " + zipDownloadSizeLimit + " bytes.\r\n");
                                     }
+                                } else if (accessService.isRestrictedByEmbargo(file.getOwner())) {
+                                    if (zipper == null) {
+                                        fileManifest = fileManifest + "File with id=" + file.getId() +
+                                                " IS EMBARGOED UNTILL "+ file.getOwner().getEmbargoDate().getOrElseThrow(() ->
+                                                new MissingParameterValueException("[Couldn't retrive embargo date]")).toInstant() + "\r\n";
+                                    } else {
+                                        zipper.addToManifest("File with id=" + file.getId() +
+                                                " IS EMBARGOED UNTILL "+ file.getOwner().getEmbargoDate().getOrElseThrow(() ->
+                                                new MissingParameterValueException("[Couldn't retrive embargo date]")).toInstant() + "\r\n");
```

Comment:
Maybe Couldn't retrieve embargo date for file id: {id}, since in logs we would need to know for which object we were not able to retrieve it.